File: timing.js
//
// Timing test
//

var a = [ ];
var n = 1000;
a.length = n;

var startTime = Date.now();

for (var i = 0; i < n; ++i) {
    for (var j = 0; j < n; ++j) {
        a[j] = j * (j + 1) / 2;
    }
}

var t = Date.now() - startTime;
print("Run time: " + (t) + "ms\n");

File: timing.lua
--
-- Timing test
--

local a = { }
local n = 1000
a.n = n

local startTime = os.clock();

for i = 0, n, 1 do
    for j = 0, n, 1 do
        a[j] = j * (j + 1) / 2;
    end
end

local t = os.clock() - startTime
print("Run time: " .. (t * 1000) .. "ms\n")
